css/sass/spacing-classes.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @margin-and-padding.scss
+ * global classes for margin and padding
+ * naming order follows the property shorthand:
+ * • top • right • bottom • left
+ * m or p (margin or padding)
+ * side  (top • right • bottom • left)
+ * amount and unit (px • em • rem)
+ * example generated classes:
+ *   .m-trbl--1px, .m-t--1em, m-b--1rem
```

Comment:
I think class names like `.m-trbl--1px` are rather obtuse. 

What is the benefit of generating all of those extra classes when you could just do: `.m-t--1px.m-b--1px` etc? That also gives you more flexibility to change it up like `.m-t--1em.m-b--2px`